cmd/juju/commands/ssh.go

```diff
@@ -28,6 +28,9 @@ The SSH host keys of the target are verified. The --no-host-key-checks option
 can be used to disable these checks. Use of this option is not recommended as
 it opens up the possibility of a man-in-the-middle attack.
 
+Options specific to ssh can be provided as well. Refer to the ssh man
+page for an explanation of those options.
+
```

Comment:
I feel like this isn't quite complete enough, because I don't think you can pass them at arbitrary points on the command line, they have to come after you specify a target. Maybe something like:

Any arguments passed after the target will be passed directly on to OpenSSH (on platforms where it is available).

Along with a later example of:

Pass extra options to SSH:
  juju ssh mysql/0 -i SPECIAL_IDENTITY_FILE echo hello

Thoughts?

The main reason is that I don't think:
 juju ssh -i IDENTITY_FILE mysql/0 
works, because the '-i' is tried to be interpreted by "juju ssh" and not being passed on to ssh.